docs/docsite/rst/network/user_guide/platform_index.rst

```diff
@@ -64,6 +64,7 @@ Settings by Platform
     `Cisco IOS`_ `[†]`_                 ``ios``                  ✓                              ✓
     `Cisco IOS XR`_ `[†]`_              ``iosxr``                ✓                              ✓
     `Cisco NX-OS`_ `[†]`_               ``nxos``                 ✓                     ✓        ✓
+    `Ciena SAOS6`_                      ``saos6``                ✓                              ✓
```

Comment:
Hi @jgroom33 and thanks for the Ansible documentation PR. Can you please move this entry above all the Cisco entries so the table stays in alphabetical order.

Reply:
good catch... yes